docs/mcp_servers.md

```diff
@@ -0,0 +1,86 @@
+# MCP Servers
+
+**PydanticAI** supports integration with
+[MCP (Model Control Protocol) Servers](https://modelcontextprotocol.io/introduction),
+allowing you to extend agent capabilities through external services. This integration enables
+dynamic tool discovery.
+
+## Install
+
+To use MCP servers, you need to either install [`pydantic-ai`](install.md), or install
+[`pydantic-ai-slim`](install.md#slim-install) with the `mcp` optional group:
+
+```bash
+pip/uv-add 'pydantic-ai-slim[mcp]'
+```
+
+!!! note
+    MCP integration requires Python 3.10 or higher.
+
+## Usage
+
+The [MCPServer][pydantic_ai.mcp.MCPServer] must be used within an async context manager to ensure
+proper initialization and cleanup of resources. You can use either use the
+[`MCPSubprocessServer`][pydantic_ai.mcp.MCPSubprocessServer] or the
+[`MCPRemoteServer`][pydantic_ai.mcp.MCPRemoteServer] class.
+
+### MCP Remote Server
+
+You can have a MCP server running on a remote server. In this case, you'd use the
+[`MCPRemoteServer`][pydantic_ai.mcp.MCPRemoteServer] class:
+
+```python {title="basic_mcp_setup.py" test="skip"}
```

Comment:
```suggestion
```python {title="mcp_remote_server.py" test="skip"}
```

Reply:
Can you suggest how? It's a remote server.